_data/retail.yml

```diff
@@ -81,13 +81,13 @@ websites:
       facebook: Costco
 
     - name: Dell
-      url: http://www.dell.com
+      url: https://www.dell.com
```

Comment:
This redirects down to HTTP. Should we have the HTTPS site listed and hope they eventually support it fully?

Reply:
I'd say yes.

Reply:
currently does not redirect for me